vmm/src/default_syscalls/filters.rs

```diff
@@ -36,14 +36,9 @@ pub fn default_filter() -> Result<SeccompFilter, Error> {
             allow_syscall(libc::SYS_brk),
             allow_syscall(libc::SYS_clock_gettime),
             allow_syscall(libc::SYS_close),
+            allow_syscall(libc::SYS_connect),
             allow_syscall(libc::SYS_dup),
-            allow_syscall_if(
-                libc::SYS_epoll_ctl,
-                or![
-                    and![Cond::new(1, Eq, super::EPOLL_CTL_ADD)?],
-                    and![Cond::new(1, Eq, super::EPOLL_CTL_DEL)?],
-                ],
-            ),
+            allow_syscall(libc::SYS_epoll_ctl),
```

Comment:
Why not filtering the epoll related operations (ADD, DEL)?

Reply:
I'm actually using `EPOLL_CTL_MOD` now for the whole `VsockEpollListener` dance, where we need to register/unregister for each individual event (EPOLLIN/EPOLLOUT).